pkg/service/test/test.go

```diff
@@ -465,17 +466,39 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp) (
 	noise := tc.Noise
 
 	var (
-		bodyNoise   []string
-		headerNoise = map[string]string{}
+		bodyNoise   map[string][]string
+		headerNoise map[string][]string
 	)
 
+	bodyNoise = map[string][]string{}
+	headerNoise = map[string][]string{}
+
 	for _, n := range noise {
 		a := strings.Split(n, ".")
 		if len(a) > 1 && a[0] == "body" {
 			x := strings.Join(a[1:], ".")
-			bodyNoise = append(bodyNoise, x)
+			bodyNoise[x] = []string{}
```

Comment:
I didn't understand this line. Can you elaborate on this..?

Reply:
i have changed the data type of bodyNoise to map[string][]string, here the key will represent the name of the noisy field and the value represents an array of regex to be ignored. So, I made these changes according to the new data type.

Reply:
understood.